code/game/objects/items/spear.dm

```diff
@@ -196,15 +196,15 @@
 	icon_prefix = "bone_spear"
 	name = "bone spear"
 	desc = "A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."
-	force = 8 //SKYRAT EDIT
+	force = 10 //SKYRAT EDIT
 	throwforce = 22
 	reach = 2 // SKYRAT EDIT
```

Comment:
the reason it's weaker, is because of reach 2, ala, it can hit from twice the range any other melee weapon could, if it was vaguely comparable to them damage wise, it'd be immediately better.

Reply:
I feel like it _should_ be immediately better than a sharp glass piece on a stick honestly. I understand caring about balance but making it worse than a bamboo stick feels a bit bad. We could always up the damage of the proper, metallic spears instead, since right now only the plastitanium one is better than whatever you can craft with wires in maintenance by the looks of it

Reply:
What's so good about the bamboo spear compared to a regular glass + rod spear?

Reply:
Oh nothing. The glass + rod spear is actually pretty good. That's the main problem, a titanium spear is literally worse unless not wielded, and only a plastitanium spear gets to its wielded force.
A bamboo one is literally on pair with titanium and can be thrown for more damage, that's what I find odd. Then again, I'm not too familiar with wound bonuses so I don't know which each receive over the other

Looking at it too, bone spears get some armor pen, so that also influences with the damage. I'm between 18 or leaving it at 16 wielded now after some deeper look at the code.